### DISCUSSION TITLE
Is it possible to open a file from Finder in a Wails app?

### DISCUSSION THREAD

**Discussion by iansinnott** [wailsapp/wails#1829] [Q&A]
**TL;DR:** I'm building an app that deals with PDFs and wondering if I can have a PDF open in Wails when double-clicked from Finder (or Explorer, etc on other platforms).

---

The open file dialog is great for telling wails to open a file. However, if the user double clicks a file I'm not sure how to have wails receive the path or file object and respond.

I suspect I can register my app for use with various file types using a custom `Info.plist`, but even so that wouldn't allow wails to handle the open event.

I did some searching around issues / docs and didn't find this info. Please excuse if I overlooked some existing discussion.

**Comment by leaanthony** [comment]
There's a ticket for this but can't find it. If you like, open a new one.

**Reply by iansinnott** [reply]
Opened: https://github.com/wailsapp/wails/issues/1830
